### ISSUE TITLE
Link.CopyElement Node returns error with 0x00000007

### ISSUE THREAD

**Issue by ShinK360** [erfajo/OrchidForDynamo#191]
### Operating system
Windows 10 Pro, Ver 10.0.19044 Build 19044

### Revit and Dynamo version
(_Which version of Revit, Dynamo and Orchid are you using? Go to Help > About if you're not sure. for Orchid version, go to Packages > Manage Packages_)
Revit: 2022.1.30.34
Dynamo core: 2.12.1.8246
Dynamo Revit: 2.12.1.8411
Orchid: 2021.7.3

### What did you do? 
I tried to copy elements from linked model and paste them on active model at the same location. Please refer to attached image for my simple script.
Then "Link.CopyElement" node returns error associated with 0x00000007, and did nothing.

Can you please have a look what causes this issue, if it relates to OS or Windows settings, I will talk to IT.

### What did you expect to see?
The element to be pasted without error!

### What did you see instead?
(Fill in here)
![Orchid 1](https://user-images.githubusercontent.com/104182852/222252207-d7817f3c-5aa9-4578-9dbc-fa030ba4a95c.png)
![Orchid 2](https://user-images.githubusercontent.com/104182852/222252232-bc6571d9-0241-4c68-a5db-5f0db23cccd7.png)

**Comment by erfajo** [comment]
Your issue is the same as another issue I got some days ago
https://github.com/erfajo/OrchidForDynamo/issues/189#

Could you please try if the "5808_beta" works for you?
- you can find it under "builds"

**Comment by ShinK360** [comment]
Thank you very much for coming back to me. Apology. I missed to find similar inquiry.

I have installed 5808 Beta, but it didn't work. 
So I uninstalled Orchid package completely, and re-installed 5808 only. 
Then Dynamo looked for 5806 pack.
I uninstalled them all again and re-installed 5806 first then 5808. 
However, Dynamo shows the package version is still 5806 and result is the same with the error massage.
5808 is in 2.16 folder and package path is set to read 2.16.

I really appreciate your hard work. Hope we can solve this together. (I have no scripting skill though but I can do whatever you tell me to test). 


![Orchid 1](https://user-images.githubusercontent.com/104182852/222541453-f4a2addb-1d81-4ceb-b400-6a642affc866.png)
![Orchid 2](https://user-images.githubusercontent.com/104182852/222541480-413d90d0-dc44-4a37-b9b4-22e083a97520.png)

**Comment by ShinK360** [comment]
Hi
I was using Revit 2022 and the node didn't work.
I tried it with Revit 2023.1.1 and it worked beautifully as I expected!!
The package shows 5808, too.

Majority of projects my company has at the moment, are still Revit 2022. It will be great if we can get it work with R22.

Thank you!!

**Comment by erfajo** [comment]
Thanks, that explained a bit :-)

I am fixing errors for all four versions in parallel, however, I only put up the beta build where I had a user that could test it...
So now I think I have nailed the error have I made new builds (essentially the same as the 5808_beta).

Starting to use the windows API is a challenge, and your error is slightly different than the other user. Will it be doable to make further tests where I continue making beta builds well-knowing that these might fail while testing which boundaries I hit?
I need to know your windows system version and your country, especially if you use other character set than Latin letters.

Otherwise, will I close this issue, thank you for your report :-)

**Comment by ShinK360** [comment]
The system details as shown below.

Operating system
Windows 10 Pro, Ver 10.0.19044 Build 19044

Revit and Dynamo version
(Which version of Revit, Dynamo and Orchid are you using? Go to Help > About if you're not sure. for Orchid version, go to Packages > Manage Packages)
Revit: 2022.1.30.34
Dynamo core: 2.12.1.8246
Dynamo Revit: 2.12.1.8411
Orchid: 2021.7.3

I am based in New Zealand and using only English as PC language.
 Again. Thank you for your great Dynamo packages!

**Comment by erfajo** [comment]
If Dynamo is 2.12, then Orchid is 212 :-)

However, your windows version is the same as the other user, so it must be something with that. I have Windows 11 and I am an administrator on my computer. Could it be so, that you are using a "work" computer where you don't have administrator rights?

When I read the documentation for the Windows API, it says that the current version is covering from the earliest windows 10, so it does not cover windows 8 and previous versions... but I didn't think anyone used that in combination with Revit anyway :-)
but I am not so fluent in the Windows API so I am aware that there are internal steps I must be aware of.
